neuralprophet/forecaster.py

```diff
@@ -1877,6 +2379,28 @@ def plot_parameters(
             # ValueError if selected quantile is out of range
             if quantile not in self.config_train.quantiles:
                 raise ValueError("Selected quantile is not specified in the model configuration.")
+        else:
+            # plot parameters for median quantile if not specified
+            quantile = self.config_train.quantiles[0]
+
+        # Validate components to be plotted
+        valid_set = [
```

Comment:
It's a bit hard to figure out the difference between components and parameters, so I'd propose be better name them more explicitly.

```suggestion
        valid_components_set = [
```